patch/cisco-Enable-static-memory-reservation-for-OIRable-PCIe-de.patch

```diff
@@ -0,0 +1,198 @@
+commit 9a57f4339767fb8129ee54e9d1714af59d4a6937
+Author: balsup <balsup@contoso.com>
+Date:   Fri Aug 14 00:09:18 2020 +0000
+
+    Reserve address space for powered-off devices behind bridges
+
+diff --git a/drivers/pci/setup-bus.c b/drivers/pci/setup-bus.c
+index 8e5b00a..062cbe1 100644
+--- a/drivers/pci/setup-bus.c
++++ b/drivers/pci/setup-bus.c
+@@ -979,6 +979,149 @@ static inline resource_size_t calculate_mem_align(resource_size_t *aligns,
+ 	return min_align;
+ }
+ 
++#define PLX_RES_MAGIC_VALUE            0xABBA
```

Comment:
hopefully, other vendors is not using this magic value which is in the cisco eeprom, so that this patch is only applicable on cisco platform.

Reply:
what if there is conflict?